person.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+class Person < Nameable
+  # accessing the attributes by attr_accessor method
+  attr_accessor :name, :age
```

Comment:
- Please Looks like you missed `attr_accessor :rentals`; Kindly check and fix it possible